Bucketeer/Sources/Internal/Remote/RequestBody/GetEvaluationsRequestBody.swift

```diff
@@ -6,4 +6,7 @@ struct GetEvaluationsRequestBody: Codable {
     let userEvaluationsId: String
     let sourceId: SourceID
     let userEvaluationCondition: UserEvaluationCondition
+    // noted: we didn't set the default value to prevent the warning about Decodeable will not work
```

Comment:
For this case, it's okay to only leave this comment on the PR.
No need to comment on the code.

Reply:
sure, let me remove it

Reply:
removed in https://github.com/bucketeer-io/ios-client-sdk/pull/48/commits/0eb8b43c08c7bc4a9feda77b9f42fcf138508110